akka-remote/src/main/scala/akka/remote/artery/ObjectPool.scala

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.remote.artery
+
+import org.agrona.concurrent.ManyToManyConcurrentArrayQueue
```

Comment:
In general I'd say "that's how you do it". The difference being between accidentally having it on the classpath because someone else pulled it in (and hoping A to keep that dependency and B that they will keep having the transitive one) vs. having it as a dependency since it is used in a project. I'd guess implicitly depending on it like this might cause troubles with OSGi as well. 

Can see the pragmatic side of it as well ofc. so, maybe when we are done with artery then.